### PR TITLE
Fixing publish action for kubenertes

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
I create a simple fix for `func azure publish` command 

## Known Issue

The command will success the deployment, however, after the deployment, when the core tools fetch the functions information that fails with 500. It is known issue. That will be fixed on the Geo master side, so that I keep the current code. 

CC @pragnagopa

```
https://management.azure.com/subscriptions/{SUBSCRIPTION_ID}/resourceGroups/LimaArcNcsEnv05/providers/Microsoft.Web/sites/tsushihttp6/functions?api-version=2015-08-01

-> Internal Server Error  (500)
```

### CODE REVIEW

**Review by pragnagopa — CHANGES_REQUESTED** [review]
support for run from package will be added soon. For existing issues - they should be fixed  as well. We should not put in temporary work-arounds

File: src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs [hunk 1]
```diff
@@ -506,6 +515,11 @@ private async Task HandleLinuxDedicatedPublish(Site functionApp, Func<Task<Strea
             await PerformAppServiceRemoteBuild(zipStreamFactory, functionApp);
         }
 
+        private async Task HandleLimaAppPublish(Site functionApp, Func<Task<Stream>> zipStreamFactory)
```
Comment (pragnagopa) [inline src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs:518]: rename to ` HandleKubeAppPublish`

Comment (pragnagopa) [reply src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs:518]: this method is not used anywhere?

**Review by pragnagopa — CHANGES_REQUESTED** [review]
added couple of comments

File: src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs (hunk 1 above)
Comment (TsuyoshiUshio) [reply src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs]: Sorry it was old method that is unused.

**Comment by TsuyoshiUshio** [comment]
Latest v3 branch already has fix.

**Pull request closed without merging**